src/gui/mainwindow.cpp

```diff
@@ -53,6 +53,9 @@
 #include "notifications.h"
 #endif
 
+#include "aboutdialog.h"
+#include "addnewtorrentdialog.h"
+#include "autoexpandabledialog.h"
```

Comment:
https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#7-header-inclusion-order
And the other similar places too.